### ISSUE TITLE
the `--format` option does not appear to work

### ISSUE THREAD

**Issue by alxndr** [codeclimate/codeclimate#685]
After talking with Ashley at Code Climate, we concluded that the CLI tool is not respecting the long command line flag `--format`. The short flag `-f` does work.

```
$ CODECLIMATE_DEBUG=1 codeclimate analyze --format text
[DEBUG] Couldn't include because part of path doesn't exist. path="./--format"
[DEBUG] Couldn't include because part of path doesn't exist. path="./text"
Starting analysis
[...snip...]
```

This is on OS X 10.11.6, Docker 1.9.1, Docker 0.5.1.

**Comment by ABaldwinHunter** [comment]
@alxndr Thanks for opening this issue! It looks we actually fixed this bug recently [here](https://github.com/codeclimate/codeclimate/commit/4201effb8c004851c302479ab6f29f1f73e440d2) but it hasn't been released yet. I'm going to go ahead and release a new version!

In the meantime, please feel free to build the image locally to use the corrected version using `make`.

**Comment by ABaldwinHunter** [comment]
@alxndr just released 0.63.7!

Could you go ahead try out the updated version?

Cheers

**Comment by alxndr** [comment]
`--format` works in 0.63.7, thanks @ABaldwinHunter !
